packages/tocco-ui/src/Popover/Popover.js

```diff
@@ -60,7 +60,7 @@ class Popover extends React.Component {
       {this.state.showToolTip && content
       && ReactDOM.createPortal(
         <Popper
-          modifiers={{preventOverflow: {enabled: false}}}
+          modifiers={{preventOverflow: {enabled: false}, hide: {enabled: false}}}
```

Comment:
remove

Reply:
I added this to prevent this console warning:
`preventOverflow` modifier is required by `hide` modifier in order to work, be sure to include it before `hide`!